src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -531,29 +530,12 @@ private void syncWrapLimit()
    
    private void updateKeyboardHandlers()
    {
-      // create a keyboard previewer for our special hooks
-      AceKeyboardPreviewer previewer = new AceKeyboardPreviewer(
-                                                         completionManager_);
-      
       // reset keyboard handlers
       widget_.getEditor().setKeyboardHandler(null);
       
       // if required add vim handlers (to main editor and our previewer)
       if (useVimMode_)
-      {
          widget_.getEditor().addKeyboardHandler(KeyboardHandler.vim());
-         previewer.addHandler(
-            new AceVimCommandHandler(new CommandWithArg<Boolean>() {
```

Comment:
Since this is the only comment I'll merge and then remove AceVimCommandHandler on my end.